src/Xrefcheck/Util.hs

```diff
@@ -65,16 +63,6 @@ posixTimeToTimeSecond posixTime =
 utcTimeToTimeSecond :: UTCTime -> Time Second
 utcTimeToTimeSecond = posixTimeToTimeSecond . utcTimeToPOSIXSeconds
 
-composeFuncList :: [a -> a] -> a -> a
-composeFuncList = foldr (.) id
-
-whenFunc :: Bool -> (a -> a) -> (a -> a)
-whenFunc True f = f
-whenFunc False _ = id
-
-unlessFunc :: Bool -> (a -> a) -> (a -> a)
-unlessFunc = whenFunc . not
-
```

Comment:
I think that @YuriRomanowski added these helper functions recently, which look useful, but I would remove them if this PR proposal is accepted because they would become unused.

I can undo this change if anyone prefers keeping them here.

Reply:
Yep, I think this is OK